lib/public/UrlChecker.js

```diff
@@ -67,21 +68,27 @@ UrlChecker.prototype.enqueue = function(url, baseUrl, customData)
 	// Undocumented internal use: enqueue(linkObj)
 	if (typeof url==="object" && url instanceof String===false && url.broken_link_checker===true)
 	{
-		return this.linkQueue.enqueue(
+		if (shouldQueryUrl(url.url.parsed.href, this) === true)
 		{
-			url: url.url.parsed,
-			data: { customData:customData, linkObj:url }
-		});
+			return this.linkQueue.enqueue(
+			{
+				url: url.url.parsed,
+				data: { customData:customData, linkObj:url }
+			});
+		}
 	}
 	// Documented use: enqueue(url, baseUrl)
 	// or erroneous and let linkQueue sort it out
 	else
 	{
-		return this.linkQueue.enqueue(
-		{
-			url: urlobj.resolve(baseUrl || "", urlobj.parse(url) ),  // URL must be absolute
-			data: { orgUrl:url, baseUrl:baseUrl, customData:customData }
-		});
+		if (shouldQueryUrl(url, this) === true)
+        {
```

Comment:
np. I have this problem with my everyday work, which forced me to use two editors. Not suggesting that you need to do this for this project.